export_test.go

```diff
@@ -148,6 +150,11 @@ func MockHashAlgAvailable() (restore func()) {
 	}
 }
 
+func ClearIsArgon2RemoteProcess() {
+	atomic.StoreUint32(&argon2RemoteProcessStatus, notArgon2RemoteProcess)
+	runtime.GC()
```

Comment:
why the GC here?

Reply:
I've added a comment to this function to explain why.

Reply:
I've removed this function (and the corresponding `SetIsArgon2RemoteProcess` entirely now.